### ISSUE TITLE
Some texts are not rendered - very rare case

### ISSUE THREAD

**Issue by mstrzoda** [wkhtmltopdf/wkhtmltopdf#3544]
Hi guys,

Just wanted to ask you if you have any idea why sometimes (very very rarly) some text are rendered and rest not, for example table is rendered with column styling, with right rows number, but without text inside. Please check the attachments
[12345.pdf](https://github.com/wkhtmltopdf/wkhtmltopdf/files/1119013/12345.pdf)
[12345-mike.pdf](https://github.com/wkhtmltopdf/wkhtmltopdf/files/1119014/12345-mike.pdf)

**Comment by daudmalik06** [comment]
@mstrzoda  yeah it happens some time try this new library which uses chrome as browser so no more these kind of issues , very simple interface , easy to install
https://github.com/daudmalik06/PhpChromeToPdf

**Comment by Tomsgu** [comment]
Do you still have this issue in 0.12.5? If yes can you provide more information?
- What os
- html minimal example
- command that you use to render it
